.githooks/pre-commit

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env sh
+RED="\033[1;31m"
+BLUE="\033[34m"
+GREEN="\033[1;32m"
+NOCOLOR="\033[0m"
+
+echo "${BLUE}(pre-commit) Cleaning compiled files...${NOCOLOR}"
+npm run clean
+
+echo "${BLUE}(pre-commit) Building backend...${NOCOLOR}"
+npm run build:backend
+
+linter_exit_code=0
+eslint_files=$(git diff --cached --diff-filter=d --name-only --line-prefix=`git rev-parse --show-toplevel`/ | grep '.js$\|.ts$\|.vue$')
+stylelint_files=$(git diff --cached --diff-filter=d --name-only --line-prefix=`git rev-parse --show-toplevel`/ | grep '.css$\|.vue$')
+
+echo "${BLUE}(pre-commit) Running linters...${NOCOLOR}"
+if [ "$eslint_files" ]
+then
+  eslint --fix $eslint_files
```

Comment:
```suggestion
  npx eslint --fix $eslint_files
```

Reply:
would the `lint:fix` script work as well? running on the changed files is certainly faster but might falsely pass in our case (e.g. if file A is deleted, but file B references A an import lint check should fail)

same for `stylelint`-ing